src/components/views/elements/MiniAvatarUploader.tsx

```diff
@@ -32,7 +32,7 @@ interface IProps {
     hasAvatar: boolean;
     noAvatarLabel?: string;
     hasAvatarLabel?: string;
-    setAvatarUrl(url: string): Promise<void>;
+    setAvatarUrl(url: string): Promise<any>;
```

Comment:
should get an `// XXX: types`, a type definition, or a comment explaining why `any` is okay.

Reply:
maybe we should alias a `DontCare` for self-documentation

Reply:
that would be best, in preparation for our `noExplicitAny: true` config

Reply:
Looks like `unknown` is the right thing to use here.